components/raftstore/Cargo.toml

```diff
@@ -10,7 +10,6 @@ publish = false
 failpoints = ["fail/failpoints"]
 prost-codec = [
   "prost",
-  "engine/prost-codec",
```

Comment:
Is there any way to enable this in the dev build?

Reply:
I am not seeing an obvious way to tie features to only dev-dependencies, and this is causing the build to fail on CI.

I think I can just remove the protobuf deps entirely from engine to get rid of the problem.

Reply:
FWIW, this was not causing the CI to fail. I pushed a fix for the CI.

I think this is harmless as-is. My main reason for thinking this is that the prost build works after this patch the same as before, including testing raftstore with the prost feature on. Another is that engine has no technical dependency on protobufs. Another is that I suspect the features are correctly enabled by other crates.

Anyway, I am still working on adding a few more commits that completely removes any direct knowledge of the prost/protobuf matter from the engine crate.